lemminx-liberty/src/test/java/io/openliberty/LibertyHoverTest.java

```diff
@@ -23,12 +25,31 @@ public void testFeatureHover() throws BadLocationException {
                                 "</server>" //
                 );
 
-                XMLAssert.assertHover(serverXML, "test/server.xml",
+                XMLAssert.assertHover(serverXML, serverXMLURI,
                                 "This feature enables support for Java API for RESTful Web Services v2.1.  "
                                                 + "JAX-RS annotations can be used to define web service clients and endpoints that comply with the REST architectural style. "
                                                 + "Endpoints are accessed through a common interface that is based on the HTTP standard methods.",
                                 r(2, 24, 2, 33));
 
         }
 
+        @Test
+        public void testXSDSchemaHover() throws BadLocationException, IOException {
+                // path to the serverXSD file in the lemminx cache
+                String serverXSDURI = "file:" + SERVER_XSD_RESOURCE.getDeployedPath().toString().replace("///", "/");
```

Comment:
should this be:
`String serverXSDURI = SERVER_XSD_RESOURCE.getDeployedPath().toUri().toString().replace("///", "/");`
or 
`String serverXSDURI = "file:" + SERVER_XSD_RESOURCE.getDeployedPath().toString()`

Reply:
Updated to use the toURI method 👍